src/crimson/os/seastore/onode_manager/staged-fltree/tree_utils.h

```diff
@@ -0,0 +1,331 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <cassert>
+#include <cstring>
+#include <random>
+#include <string>
+#include <sstream>
+#include <utility>
+#include <vector>
+
+#include "crimson/common/log.h"
+#include "stages/key_layout.h"
+#include "tree.h"
+
+namespace crimson::os::seastore::onode {
+
+ghobject_t make_ghobj(
+    int shard, unsigned pool, unsigned crush,
+    std::string ns, std::string oid, unsigned snap, unsigned gen) {
```

Comment:
Isn't there already a ghobject constructor for this?  If not, you should probably make this a static method/function in that file.

Reply:
added a constructor, fixed in https://github.com/ceph/ceph/pull/36986/commits/0fab43b9b0e6b0cac620b03604ab2fad8a61121d